bom/runtime/pom.xml

```diff
@@ -1712,6 +1713,12 @@
                 <artifactId>neo4j-java-driver</artifactId>
                 <version>${neo4j-java-driver.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>${mockito.version}</version>
+                <scope>test</scope>
```

Comment:
One last minor comment: we don't want to define the scope in the bom. We did that at the beginning and we had people complaining about it.

Once it's fixed, this can go in.

Thanks!

Reply:
No problem, it's fixed.